Makefile-az.mk

```diff
@@ -232,6 +232,9 @@ az-rmnodeclaims-fin: ## Remove Karpenter finalizer from all nodeclaims (use with
 az-rmnodeclaims: ## kubectl delete all nodeclaims; don't wait for finalizers (use with care!)
 	kubectl delete --wait=false nodeclaims --all
 
+az-taintnodes: ## Run e2etests
+	kubectl taint nodes CriticalAddonsOnly=true:NoSchedule --all --overwrite
```

Comment:
Do you have use cases where this is not constrained to system nodepool? If not - should we target it explicitly with selector?

Reply:
I use this sometiems when running perf test, but not all of the nodes have scaled down I just want to launch a bunch of small instances to test somethign. 

So tainting all nodes is useful

Reply:
maybe tainting by label could be a second command?

Reply:
That's a fair idea. I've been running it right after cluster creation. However, it would be helpful to ensure it only targets system nodepool. I'll look at doing that.

Reply:
Will need a follow up PR though as I already completed